packages/components/src/ui/context/test/wordpress-component.tsx

```diff
@@ -14,6 +14,7 @@ import { forwardRef } from '@wordpress/element';
 import type { WordPressComponentProps } from '../wordpress-component';
 
 // Static TypeScript checks
+/* eslint-disable jest/expect-expect */
```

Comment:
This contains static typescript tests, so intends to use TS for checking for failures, and not Jest. While I have mixed feelings about this manifestation of tests, this intentionally doesn't contain assertion functions, so we're intentionally disabling the rule here.